src/rai/rai/tools/ros/native.py

```diff
@@ -68,6 +68,7 @@ class PubRos2MessageToolInput(BaseModel):
 
 # --------------------- Tools ---------------------
 class Ros2BaseTool(BaseTool):
+    # TODO: Make the decision between rclpy.node.Node and RaiNode
```

Comment:
FYI: using `RaiNode` here causes cicrular import error

Reply:
Yeah, had that problem. Importing under
```python
if TYPECHECKING:
 import ... RaiNode
 
# and then
node: "rclpy.node.Node | RaiNode"
 ```
 Does not work either. Nonetheless RaiNode is better suited here.